Cargo.toml

```diff
@@ -27,19 +27,18 @@ num_cpus = "1.0"
 num-format = "0.4"
 rand = "0.8"
 regex = "1"
-reqwest = { version = "0.11",  default-features = false, features = [
+reqwest = { version = "0.11", default-features = false, features = [
     "cookies",
     "gzip",
     "json",
 ] }
-serde = { version = "1.0", features = [
-    "derive",
-] }
+serde = { version = "1.0", features = ["derive"] }
 serde_cbor = "0.11"
 serde_json = "1.0"
 simplelog = "0.10"
 strum = "0.24"
 strum_macros = "0.24"
+tdigest = { version = "0.2.3", features = ["serde", "use_serde"] }
```

Comment:
It looks like this library hasn't been updated in 2 years, is there anything that has replaced it? Does it matter that it's not being updated?
https://github.com/MnO2/t-digest

Reply:
Good question, there is another implementation here https://docs.rs/pdatastructs/latest/pdatastructs/tdigest/struct.TDigest.html however that doesn't implement serde serialize/deserialize which we would need. 

I am not concerned about the crate not getting updates as this is the kind of things that once implemented correctly doesn't need ongoing maintanence. However if you would prefer I can change out the implementation. 

Or as we discussed I could make the implementation generic and then users can bring their own solution.